src/core/worker.js

```diff
@@ -436,6 +437,55 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return obj;
```

Comment:
A tiny addition: usually we write this with spaces after `{` and before `}`, so `{ name: obj.name }` would be the preferred notation. See also the other object notations in this PR and file.